events/views.py

```diff
@@ -1,3 +1,26 @@
-# from django.shortcuts import render
+from django.shortcuts import render, redirect
+from events.forms import EventCreationForm
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+REDIRECT_PAGE = 'welcome_page'
+
+
+@login_required(login_url=REDIRECT_PAGE)
+def create_event(request):
+
+    if request.method == 'POST':
+        print(request.POST)
```

Comment:
Can you remove this line.
This line is for debugging purposes only.

Reply:
done